### DISCUSSION TITLE
Official Project URL(s)?

### DISCUSSION THREAD

**Discussion by russellvt** [russellvt/pnp4nagios#9] [General]
Starting  this discussion from Issues #5 from @r-linder. Please feel free to resume discussion, here.

**Comment by tjyang** [comment]
What is next ? 
Looks to me there are two options.
- Ask @lingej  to add @russellvt as new admin and use original project path.
- Create a github organization to host the revived project.

**Comment by lingej** [comment]
Hi !

the domain pnp4nagios.org is paid until 12.2023
i can send you an auth codo to allow the domain transfer.

Please contact me directly via email

Best regards 

Joerg Linge

**Comment by tjyang** [comment]
@russellvt and all

Hope you all have a good holiday breaks.  
I like to resume the discussion of how can we as a team to keep pnp4nagios going as an valuable add-on to nagiosocore. 

-  @lingej transferred the pnp4nagios.org to me and I re-enabled [docs.pnp4nagios.org](http://docs.pnp4nagios.org) site using a free-tier OCI instance. 
-   [pnp4nagios organization](https://github.com/pnp4nagios)  is used  for easier project hand-over.  
     - Let me know if you like to help and be added into this organization. 
     - We need more people know php coding since I myself don't.
-  [Roadmap](https://github.com/orgs/pnp4nagios/projects/1/views/1) 
     -  Porting pnp4nagios code to work on php7/8.
     -  Address CVE issues against pnp4nagios.
-  I like to suggest moving the discussion to https://github.com/pnp4nagios/pnp4nagios/discussions.
